### PR TITLE
Update minimal websocket client example.

### LINKED ISSUE #6631: Outdated minimal example in websocket page
**Your Godot version:** 4.0 beta 13

**Issue description:** Minimal example in main websocket page is outdated.

**URL to the documentation page:** https://docs.godotengine.org/en/stable/tutorials/networking/websocket.html?highlight=websocket#minimal-client-example

### LINKED ISSUE #9138: Outdated Documentation : WebSocketClient became WebSocketPeer
**Your Godot version:**
4.2 
**Issue description:**
The Page for deprecated WebSocketClient is still top research on Google and doesn't mention that WebSocketPeer is now in use.

**Fix**
- a clear indication that WebSocketPeer is the new standard.

- a stronger visual clue that the content is deprecated in 4.2 (the green text box on top is easy to miss since it's also used to indicate that the content is actually up to date, using orange would be better to warn the user)

### PR DESCRIPTION
**Pull request by aGuyWhoMadeGames**
The new example is a combination of the old example and the example from the class reference.
This fixes #9138 and  fixes #6631.

### CODE REVIEW

File: tutorials/networking/websocket.rst [hunk 1]
```diff
@@ -36,60 +36,62 @@ This example will show you how to create a WebSocket connection to a remote serv
...
+	func _process(_delta):
+		# Call this in _process or _physics_process. Data transfer and state updates
+		# will only happen when calling this function.
+		socket.poll()
+		
+		# get_ready_state() tells you what state the socket is in.
+		var state = socket.get_ready_state()
+		
+		# WebSocketPeer.STATE_OPEN means the socket is connected and ready
+		# to send and receive data
```
Comment (AThousandShips) [inline tutorials/networking/websocket.rst:69]:
```suggestion
		# to send and receive data.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).

File: tutorials/networking/websocket.rst [hunk 2]
```diff
@@ -36,60 +36,62 @@ This example will show you how to create a WebSocket connection to a remote serv
...
+		if state == WebSocketPeer.STATE_OPEN:
+			while socket.get_available_packet_count():
+				print("Got data from server: ", socket.get_packet().get_string_from_utf8())
+		
+		# WebSocketPeer.STATE_CLOSING means the socket is closing.
+		# It is important to keep polling for a clean close
```
Comment (AThousandShips) [inline tutorials/networking/websocket.rst:75]:
```suggestion
		# It is important to keep polling for a clean close.
```

File: tutorials/networking/websocket.rst [hunk 3]
```diff
@@ -36,60 +36,62 @@ This example will show you how to create a WebSocket connection to a remote serv
...
-
-    func _process(delta):
-        # Call this in _process or _physics_process. Data transfer, and signals
-        # emission will only happen when calling this function.
-        _client.poll()
-
-This will print:
+	extends Node
+
+	# The URL we will connect to
```
Comment (AThousandShips) [inline tutorials/networking/websocket.rst:41]:
```suggestion
	# The URL we will connect to.
```

File: tutorials/networking/websocket.rst [hunk 4]
```diff
@@ -36,60 +36,62 @@ This example will show you how to create a WebSocket connection to a remote serv
...
+	@export var websocket_url = "wss://echo.websocket.org"
+
+	# Our WebSocketClient instance
```
Comment (AThousandShips) [inline tutorials/networking/websocket.rst:44]:
```suggestion
	# Our WebSocketClient instance.
```

File: tutorials/networking/websocket.rst [hunk 5]
```diff
@@ -36,60 +36,62 @@ This example will show you how to create a WebSocket connection to a remote serv
...
+		# It is now safe to stop polling.
+		elif state == WebSocketPeer.STATE_CLOSED:
+			# The code will be -1 if the disconnection was not properly notified by the remote peer.
+			var code = socket.get_close_code()
+			print("WebSocket closed with code: %d. Clean: %s" % [code, code != -1])
+			set_process(false) # Stop processing.
+
+
+
+
```
Comment (AThousandShips) [inline tutorials/networking/websocket.rst:89]: Too many empty lines, just leave two

**Comment by aGuyWhoMadeGames** [comment]
Thanks for the corrections. This is my first PR other than a few small fixes, so I wasn't aware of all of the guidelines. I fixed everything you said.

**Review by skyace65 — APPROVED** [review]
Code works for me

File: tutorials/networking/websocket.rst [hunk 6]
```diff
@@ -36,60 +36,60 @@ This example will show you how to create a WebSocket connection to a remote serv
...
+		elif state == WebSocketPeer.STATE_CLOSING:
+			pass
+		
+		# WebSocketPeer.STATE_CLOSED means the connection has fully closed.
+		# It is now safe to stop polling.
+		elif state == WebSocketPeer.STATE_CLOSED:
+			# The code will be -1 if the disconnection was not properly notified by the remote peer.
+			var code = socket.get_close_code()
+			print("WebSocket closed with code: %d. Clean: %s" % [code, code != -1])
+			set_process(false) # Stop processing.
```
[the remark this answers is not in the document]
Comment (aGuyWhoMadeGames) [reply tutorials/networking/websocket.rst]: I converted the tabs to spaces. If it matters, shouldn't this be part of the lint though?

**Review by AThousandShips — APPROVED** [review]

File: tutorials/networking/websocket.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (AThousandShips) [reply tutorials/networking/websocket.rst]: I'm not sure why it isn't but I think it's a limitation of the system, unsure

**Comment by AThousandShips** [comment]
Thanks!

**Comment by Faless** [comment]
Not sure why the PR changed the websocket address, but we moved away from
the one used in this PR because it was very unreliable (i.e. was offline
for a very long time).

**Comment by aGuyWhoMadeGames** [comment]
I changed the url because the previous one wasn't working, and all of the minimal examples I saw used the one I'm using here.

**Pull request merged**